src/map/status.cpp

```diff
@@ -2239,16 +2239,17 @@ bool status_check_skilluse(struct block_list *src, struct block_list *target, ui
  * Checks whether the src can see the target
  * @param src:	Object using skill on target [PC|MOB|PET|HOM|MER|ELEM]
  * @param target: Object being targeted by src [PC|MOB|HOM|MER|ELEM]
+ * @param checkblind: Whether blind condition should be considered (sets view range to 1)
  * @return src can see (1) or target is invisible (0)
```

Comment:
If we already touch the function signature, it would be good to change the return type from int32 to bool

Reply:
Done